packages/contracts/contracts/optimistic-ethereum/OVM/bridge/messaging/OVM_L1CrossDomainMessenger.sol

```diff
@@ -56,6 +56,7 @@ contract OVM_L1CrossDomainMessenger is
      **********************/
 
     mapping (bytes32 => bool) public blockedMessages;
+    address public customRelayer;
```

Comment:
if I understood @souradeep-das we're not planing to use the CDM changed contract but a the separate one?

Reply:
yes, should we remove the changes to the original CDM and make the LP work with normal messengers for now? in anticipation of integrating with the fast relayer in the next PR.

Reply:
Yes, the changes to the original CDM are temporary. Suggest to (1) merge this PR with the CDM changes so that everything works, (2) test the new message-relayer-fast (which would include the CDM reversion), and then (3) apply that PR in one fell swoop